Themes/default/languages/LightPortal/.spanish_es.php

```diff
@@ -54,25 +57,44 @@
 $txt['lp_show_debug_info_subtext'] = '¡Esta información estará disponible solo para administradores!';
 $txt['lp_cache_update_interval'] = 'El intervalo de actualización del caché';
 
+// Pages and blocks
 $txt['lp_extra'] = 'Páginas y bloques';
 $txt['lp_extra_info'] = 'Aquí puede encontrar configuraciones generales de páginas y bloques.';
 
 $txt['lp_show_tags_on_page'] = 'Mostrar palabras clave en la parte superior de la página';
 $txt['lp_show_comment_block'] = 'Mostrar bloque de comentarios';
-$txt['lp_show_comment_block_set'] = array('none' => 'Ninguno', 'default' => 'Integrado');
-$txt['lp_num_comments_per_page'] = 'Número de comentarios de los padres por página'; // Check meaning
-$txt['lp_page_editor_type_default'] = 'El tipo de editor de la página por defecto';
+$txt['lp_disabled_bbc_in_comments'] = 'BBC permitidos en los comentarios';
+$txt['lp_show_comment_block_set'] = array('ninguno' => 'Ninguno', 'por defecto' => 'Integrado');
```

Comment:
Please, do not translate array keys, it's a bad idea.

Reply:
You are right I did not realize, I modify it!